lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word
+
+  def initialize(word)
+    @word = word
+  end
+
+  def score
+    @point_values(letter)
+        letters = letter.chars.map.upcase do |char|
```

Comment:
It looks like you were thinking of useful methods but the order is a bit off. This would have worked if you used letters.upcase.chars.map do |char|